packages/walletconnect/src/v1.ts

```diff
@@ -13,7 +13,7 @@ function walletConnect(
   options: WalletConnectOptions = { version: 1 }
 ): WalletInit {
   const {
-    bridge = 'https://bridge.walletconnect.org',
+    bridge = 'https://derelay.rabby.io',
```

Comment:
We will not be able to support this change as a default. This package will be marked as deprecated when the WC team shuts down support for v1 but Dapps can choose to pass this bridge URL in if they would like.